### DISCUSSION TITLE
Δήλωση πρόθεσης προαιρετικής παρουσίασης μπόνους

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#173] [Show and tell]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση πρόθεσης της προαιρετικής παρουσίασης της προόδου στην ΓΕ2 στέλνοντας το τελικό αρχείο παρουσίασης

**Comment by tpanagioto** [comment]
ΟΜΑΔΑ 1:
ΓΙΑΝΝΑΚΑΚΟΥ
ΠΑΓΙΑΤΗΣ
ΠΑΝΑΓΙΩΤΟΠΟΥΛΟΣ
ΠΑΠΑΔΟΠΟΥΛΟΣ


ΓΙΑ ΠΑΡΟΥΣΙΑΣΗ ΠΡΟΟΔΟΥ ΤΗΣ ΕΡΓΑΣΙΑΣ ΣΔΥ60 - ΓΕ2 ΣΤΗΝ ΟΣΣ ΤΗΝ 01-12-2024

**Comment by tpanagioto** [comment]
Παρουσίαση της ομάδας 1
[ΣΔΥ 60 – ΣΧΕΔΙΑΣΗ ΚΑΙ ΑΝΑΛΥΣΗ ΥΛΙΚΟΛΟΓΙΣΜΙΚΟΥ.pptx](https://github.com/user-attachments/files/18053255/60.pptx)
